botocore/response.py

```diff
@@ -80,6 +80,51 @@ def read(self, amt=None):
             self._verify_content_length()
         return chunk
 
+    def __iter__(self):
+        """Return an iterator to yield lines from the raw stream.
+        """
+        # Reading 1024 bytes at a time seems a sane choice [citation needed].
+        # If the user needs to specify something different they can always call
+        # streaming_body.iter_lines(<other-chunk-size>)
+        default_chunk_size = 1024
+        return self.iter_lines(default_chunk_size)
+
+    def iter_lines(self, chunk_size):
```

Comment:
I think I would prefer to have this as a private method, normal file objects do not have an ``iter_lines`` method.